platform.md

```diff
@@ -148,6 +159,441 @@ A platform MAY support any number of mixins for a given stack in order to suppor
+      - `run-image.top-layer` SHALL be set to the uncompressed digest of the top layer in `<run-image>`
+
+#### `launcher`
+Usage:
+```
+/cnb/lifecycle/launcher [--] [<cmd> <arg>...]
+```
+
+| Input               | Environment Variable  | Default Value         | Description
+|---------------------|-----------------------|-----------------------|---------------------------------------
+| `<app>`             | `CNB_APP_DIR`         | `/workspace`          | Path to application directory
+| `<layers>`          | `CNB_LAYERS_DIR`      | `/layers`             | Path to layer directory
+| `<process-type>`    | `CNB_PROCESS_TYPE`    | `web`                 | `type` of process to launch
+| `<direct>`          |                       | `false`               | Execution strategy for user provided process
+| `<cmd>`             |                       |                       | User-provided command
+| `<arg>...`          |                       |                       | Arguments to user-provided command
+| `<layers>/config/metadata.toml`    |        |                       | Build metadata (see [`metadata.toml`](#metdata.toml-(toml))
+| `<layers>/<buildpack-id>/<layer>/` |        |                       | Launch Layers
+
+- IF `$1` is `--` `<direct>` is `true` and `<args>` SHALL be `${@2:}`
+- IF `$1` is anything other than `--`, `<direct>` is `false`, and `<args>` SHALL be `$@`
+- IF `<direct>`
+- IF `<args>` are NOT provided `launcher` SHALL select the process with `type` equal to `<process-type>` from `<layers>/config/metadata.toml` and launch it as described in the [Buildpack Interface Specification](buildpack.md).
+- IF `<args>` are provided `launcher` SHALL behaves as if it selected a process with the following process from `metadata.toml`.
+```
+[[process]]
+direct = <direct>
+command = "<cmd>"
+args = ["<arg>"...]
+```
+
+- The lifecycle SHOULD replace the lifecycle process in memory without forking it.
+- The process execution environment SHALL inherit all variables from lifecycle execution environment, with the following exceptions:
+    - `CNB_APP_DIR` SHALL NOT be set in the process environment
+    - `CNB_LAYERS_DIR` SHALL NOT be set in the process environment
+    - `CNB_PROCESS_TYPE` SHALL NOT be set in the process environment
+
+### Run Image Resolution
+
+Given stack metadata containing `run-image.image` and a set of `run-image.mirrors`. The `<run-image>` for a given `<image>` shall be resolved as follows:
+- IF any of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, this value will become the `<run-image>`
+- IF none of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, `<run-image.image>` will become the `<run-image>`
+
+### Registry Authentication
+
+The platform MAY set `CNB_REGISTRY_AUTH`  in the lifecycle execution environment, where value of `CNB_REGISTRY_AUTH`, MUST be valid JSON object and MAY contain any number of `<regsitry>` to `<auth-header>` mappings.
```

Comment:
I think the second comma (after "where value of `CNB_REGISTRY_AUTH`) is unnecessary.